.env

```diff
@@ -0,0 +1,12 @@
+# Auth Providers
+GOOGLE_CLIENT_ID="531113016303-j4fjfj2hftgrisstov4f8ub3ari723h4.apps.googleusercontent.com"
```

Comment:
.env file shouldn't be pushed into a repo, all variables should be set at Vercel console as well